default/stringtables/en.txt

```diff
@@ -1915,6 +1915,9 @@ Automatically quits once any turns specified by --auto-advance-n-turns are compl
 OPTIONS_DB_BINARY_SERIALIZATION
 Use Binary serialization for saving games. Binary serialization is faster to save and load, but may not be possible to load on a different operating system.
 
+OPTIONS_DB_SERVER_BINARY_SERIALIZATION
+The server will use Binary serialization for client-server interaction if the client's version match the server's version. Binary serialization is faster to transfer, but may not be compatible between different operating systems and architectures.
```

Comment:
One least tweak needed here, for proper verb form:
"if the client's version match" --> "if the client's version matches"

Reply:
Fixed.